src/GslCore/CommandConfig.fs

```diff
@@ -5,8 +5,8 @@ open commonTypes
 open pragmaTypes
 open Amyris.Bio.utils
 
-let v = System.Reflection.Assembly.GetExecutingAssembly().GetName().Version
-let version = sprintf "%d.%d.%d" v.Major v.Minor v.Build

```

Comment:
For some weird reason `.GetName().Version` doesn't reflect the attributes being automatically set by fake in assemblyInfo.fs. This change fixes that along with extracting the git sha as well to be very precise about current version.